scale_build/utils/manifest.py

```diff
@@ -159,3 +161,23 @@ def update_packages_branch(branch_name):
 
     with open(MANIFEST, 'w') as f:
         f.write(updated_str)
+
+
+def validate_manifest():
+    manifest = get_manifest()
+    if SKIP_SOURCE_REPO_VALIDATION:
+        return
+
+    # We would like to make sure that each package source we build from is from our fork and not another one
+    invalid_packages = []
+    for package in manifest['sources']:
+        url = urlparse(package['repo'])
+        if 'github.com' != url.hostname.strip('www.') or not url.path.startswith('/truenas/'):
```

Comment:
`str.strip` does not work this way
```suggestion
        if url.hostname not in ['github.com', 'www.github.com'] or not url.path.startswith('/truenas/'):
```